tags/tags.go

```diff
@@ -3,28 +3,27 @@ package tags
 import (
 	"strings"
 
-	"github.com/google/martian/v3/tags"
 	"github.com/pkg/errors"
 )
 
-type Key = tags.Key
+type Key = string
```

Comment:
The martian code (added by you) was

```
type Key string
```

shouldn't this be a type definition, not an alias, so that we retain a non-string type?

Reply:
Yeah, I missed that it was a typedef, not an alias. In 9abc4fe.